.travis.yml

```
@@ -26,4 +40,15 @@ after_script:
     - printf "$TRAVIS_COMMIT_LOG\n"
 
 after_success:
-    - if [[ ! -z "${CODECOV_TOKEN}" ]]; then (bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"); fi;
+    - |
+      if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
+          if [[ "${TRAVIS_COMPILER} == "gcc" "]]; then
```

Redundant quote too?
```suggestion
          if [[ "${TRAVIS_COMPILER} == "gcc" ]]; then
```